data/assets/scene/solarsystem/heliosphere/todayssun/whiteBackground.asset

```diff
@@ -0,0 +1,46 @@
+local transforms = asset.require("scene/solarsystem/sun/transforms")
```

Comment:
The filename should be `whitebackground.asset` or `white_background.asset` since we only use lowercase letters for filenames